### ISSUE TITLE
Add support for streamWrapper::stream_set_option()

### ISSUE THREAD

**Issue by mikey179** [bovigo/vfsStream#15]
vfsStream should support `streamWrapper::stream_set_option()` so it can support `stream_set_blocking()`, `stream_set_timeout()` and `stream_set_write_buffer()`.

**Comment by kiler129** [comment]
Did you plan to support that in near future? :)

**Comment by mikey179** [comment]
No, currently I have no plans for this, as I'm missing both the time and need for doing this. So any PR is welcome.
